vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java

```diff
@@ -535,7 +544,14 @@ public VaadinServletContextInitializer(ApplicationContext context) {
         }
 
         String onlyScanProperty = appContext.getEnvironment()
-                .getProperty("vaadin.whitelisted-packages");
+                .getProperty("vaadin.allowed-packages");
+        if (onlyScanProperty == null) {
+            onlyScanProperty = appContext.getEnvironment()
+                    .getProperty("vaadin.whitelisted-packages");
+            if (onlyScanProperty != null) {
+                getLogger().warn("vaadin.whitelisted-packages is deprecatd. Use vaadin.allowed-packages instead.");
```

Comment:
I would use a message similar to the one used for `blacklisted-packages
```suggestion
                getLogger().warn("vaadin.whitelisted-packages is deprecatd and may not be supported in the future. Use vaadin.allowed-packages instead.");
```